ui/.eslintrc

```diff
@@ -10,6 +10,7 @@
     "plugin:@typescript-eslint/recommended"
   ],
   "rules": {
-    "no-debugger": 0
+    "no-debugger": 0,
+    "@typescript-eslint/no-explicit-any": ["off"]
```

Comment:
I had added this to turn off eslint on any types, which are not avoidable when we are making fetch calls. I will consult with Shakira on this to see if there is a better way to make the types not be `any`